aten/src/ATen/test/complex_test.cpp

```diff
@@ -0,0 +1,33 @@
+#include <gtest/gtest.h>
+#include "c10/util/Complex.h"
+
+template<typename T, typename int_t>
+static void TestIntegerOp(T real, T img, int_t i) {
```

Comment:
I think we should change the name of this function to TestBinaryOps for clarity

Reply:
```
template<typename T, typename int_t>
static void TestBinaryOps(T real, T img, int_t num) {
...
}
```
and change i --> num to avoid it being confused for imaginary number) as I mentioned before

Reply:
I redid both names for clarify. Please check